pom.xml

```diff
@@ -4,7 +4,7 @@
     <groupId>com.adyen</groupId>
     <artifactId>adyen-java-api-library</artifactId>
     <packaging>jar</packaging>
-    <version>2.2.0</version>
+    <version>2.2.1-SNAPSHOT</version>
```

Comment:
Version upgrade not needed from this pull request since it will be taken care by Adyen team. Please remove this change.